pkg/gds/store/store.go

```diff
@@ -95,15 +96,24 @@ type DirectoryStore interface {
 	DeleteVASP(id string) error
 }
 
-// CertificateStore describes how the service interacts with Certificate requests.
-type CertificateStore interface {
-	ListCertReqs() iterator.CertificateIterator
+// CertificateRequestStore describes how the service interacts with Certificate requests.
+type CertificateRequestStore interface {
```

Comment:
I suppose I thought this name change would reach out and touch a lot of code but I guess the DB just implements all these interfaces so it's really just the inside of the `store` package that needs renaming - big win!